doc/rst/technotes/gpu.rst

```diff
@@ -261,34 +220,77 @@ For more examples see the tests under |multi_locale_dir|_ available from our `pu
 Memory Strategies
 ~~~~~~~~~~~~~~~~~
 
-Currently by default Chapel uses unified memory feature to store data that is
-allocated on a GPU sublocale (i.e. ``here.gpus[0]``).  Under unified memory the
-CUDA driver implicitly manages the migration of data to and from the GPU as
-necessary.
-
-We provide an alternate memory allocation strategy that stores array data
-directly on the device and store other data on the host.  There are multiple
-benefits to using this strategy including that it enables users to have more
-explicit control over memory management, may be required for Chapel to
-interoperate with various third-party communication libraries, and may be
-necessary to achieve good performance. As such it may become the default memory
-strategy we use in the future. Be aware though that because this strategy is
-relatively new addition it hasn't been as thoroughly tested as our
-unified-memory based approach.
-
-To use this new strategy set the environment variable ``CHPL_GPU_MEM_STRATEGY``
-to ``array_on_device``.  For more examples that work with this strategy see the
-tests under |page_lock_mem_dir|_  available from our `public Github repository
-<https://github.com/chapel-lang/chapel>`_.
-
-.. |page_lock_mem_dir| replace:: ``test/gpu/native/page-locked-mem/``
-.. _page_lock_mem_dir: https://github.com/chapel-lang/chapel/tree/main/test/gpu/native/page-locked-mem
+The ``CHPL_GPU_MEM_STRATEGY`` environment variable can be used to choose between
+two different memory strategies.
+
+The current default strategy is ``unified_memory``. This strategy stores data
```

Comment:
I think this sentence might be a little misleading in that folks might assume storing data on a GPU sublocale means storing data on the GPU:

```
This strategy stores data that is allocated on a GPU sublocale (i.e. ``here.gpus[0]``).
```

Maybe say "applies to all" instead:

```
The strategy applies to all data allocated on a GPU sublocale (i.e. ``here.gpus[0]``)
```